security/pkg/workload/server.go

```diff
@@ -61,12 +49,21 @@ type SDSServer struct {
 
 	// Read/Write mutex for privateKey
 	privateKeyGuard sync.RWMutex
+
+	// Specifies a map of Unix Domain Socket paths and the server listens on.
+	// Each UDS path identifies the identity for which the workload will
+	// request X.509 key/cert from this server. This path should only be
+	// accessible by such workload.
+	udsServerMap map[string]*grpc.Server
+
+	// Mutex for udsServerMap
+	udsServerMapGuard sync.Mutex
+
+	// current certificate chain and private key version number
+	version string
```

Comment:
What is this version used for?

Reply:
DiscoveryResponse response requires version_info to check certificates were updated.
When the version increased, subscription module of the Envoy calls onConfigUpdate function.

https://github.com/envoyproxy/data-plane-api/blob/78b7960a01bd1ad5248cb8de230ddb04ccf6dfd4/envoy/api/v2/discovery.proto#L60